### DISCUSSION TITLE
Allergies card

### DISCUSSION THREAD

**Discussion by Ayrton31** [piitaya/lovelace-mushroom#406] [Ideas]
Hello,

What do you think by creating a card perhaps based on the same look than this one : [https://github.com/piitaya/lovelace-mushroom/pull/388](url) to monitor allergies in a city. I think that 4 allergies could be enough to keep a capacity to read information (like exemple for plants)
It's quite simple to retrieve on different sites the value of allergies (based on a metric).

On a stock card "ha-card-weather-conditions" if found this capacity but the look it's too different than Mushroom one:
![image](https://user-images.githubusercontent.com/73702716/165480948-196c3a31-e4a1-43da-b196-2a22a08bcef6.png)

In addition of the visual rendering it would be important to manage the ratio. Indeed, depending on the site you retrieve datas the allergies could be 2 but the maximum could be 3 or 2 and the maximum could be five.

What do you think about that ?

**Comment by piitaya** [comment]
The [weather entity](https://developers.home-assistant.io/docs/core/entity/weather/) does not return this sensor data. It's returned to separate entity. I think this usage is too advanced. 
It's better for now to add chip or card for this.

**Reply by Ayrton31** [reply]
Yes clearly it's a separate sensor but Climacell (Tomorrow.io) provide this informations.
Hum.... I'm not convinced that using chip card will be a good idea. It's not very visual but information very relevant for allergies person ;)
